conf/public_aws_ecr.config

```diff
@@ -0,0 +1,54 @@
+/*
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+    AWS ECR Config
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+    Config to set public AWS ECR images wherever possible
+    This improves speed when running on AWS infrastructure.
+    Use this as an example template when using your own private registry.
+----------------------------------------------------------------------------------------
+*/
+
+docker.registry = 'public.ecr.aws'
+podman.registry = 'public.ecr.aws'
+
+process {
+    withName: '.*:CAT_FASTQ' {
+        container = 'quay.io/nf-core/ubuntu:20.04'
+    }
```

Comment:
Nice! I say we go with the Docker container on nf-core because that is the exact version we use in the pipeline too. We have seen weird issues in the past where tagged ubuntu containers have been updated 😱

Reply:
In that case, can I get an approval please so we can test?